app/org/sagebionetworks/bridge/dynamodb/DynamoNotificationRegistration.java

```diff
@@ -13,7 +13,6 @@
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
 @DynamoDBTable(tableName = "NotificationRegistration")
-@BridgeTypeName("NotificationRegistration")
```

Comment:
It makes more sense to me to put these on the interface; this is part of the public interface of the object.